### ISSUE TITLE
Can't log in after adding authorization code until AuthorizeRouteView is added

### ISSUE THREAD

**Issue by ddashwood** [dotnet-presentations/blazor-workshop#240]
In chapter 6, the instructions talk you through adding the authorization code. It then says that you should run the program, register, and login. But after logging in, an exception occurs (`Cannot read property 'completeSignIn' of undefined`)

To fix this, simply follow the rest of the instructions. When it gets to the point where you add an AuthorizeRouteView, the program starts working again.

I don't understand why it requires a AuthorizeRouteView, since it's not trying to display a route that requires authorization. Perhaps someone could explain why this is happening? But regardless of the reason, I'd suggest that the instructions need to be changed to use the AuthorizeRouteView much earlier on, before attempting to register a user and login. Unless I've done something wrong, of course.

**Comment by OrihuelaConde** [comment]
The console error is: Unhandled exception rendering component: Cannot read property 'completeSignIn' of undefined

**Comment by ddashwood** [comment]
> The error in the console is: Unhandled exception rendering component: Cannot read property 'completeSignIn' of undefined

You are quite right. I’ve edited my comment to include the correct error message. Thanks
